python/tvm/relay/frontend/tensorflow2.py

```diff
@@ -57,6 +57,27 @@ def _infer_type_with_prelude(val, prelude):
 def _need_prelude_for_shape_inference(op):
     return "TensorList" in op or "TensorArray" in op
 
+def detect_tf2_control_flow(_graph):
+    tensorlist_ops = ["TensorListFromTensor", "TensorListGetItem",  "TensorListReserve", "TensorListSetItem", "TensorListStack"]
```

Comment:
I don't think tensorlist ops need to be tied specifically to TF2 control flow. Although they are often used together, what do you think about removing this condition? Do the tensorlist tests still work using the original parser?

Reply:
The `test_sequential.py` tests for `tensorlist_stack` and `tensorlist_read` fail with similar messages like below. However I do agree that these are not control_flow ops and the name should be changed.

> The following operators are not implemented: {'TensorListGetItem', 'TensorListSetItem', 'TensorListReserve'}